src/main/java/com/stripe/model/SubscriptionScheduleRevisionCollection.java

```diff
@@ -0,0 +1,56 @@
+package com.stripe.model;
+
+import com.stripe.Stripe;
+import com.stripe.exception.StripeException;
+import com.stripe.net.ApiResource;
+import com.stripe.net.RequestOptions;
+import java.util.Map;
+
+public class SubscriptionScheduleRevisionCollection
```

Comment:
Hmm.. actually no other resource returns this collection, so we can never get an instance of this collection to make the API request (with the existing url)..

Reply:
I think we can go with the followings..
In `SubscriptionSchedule.java` we have
`public SubscriptionScheduleRevisionCollection revisions() `
Then, we remove the `list` method from the `SubscriptionScheduleRevisionCollection.java`..
I fixed my client metadata and the generated outcome looks like this..
https://github.com/stripe/stripe-java/blob/abaada5f48449f495886efdda0eed0c5583bbe0b/src/main/java/com/stripe/model/SubscriptionSchedule.java#L338-L356


I believe this is similar to account/person relationship

Reply:
@mickjermsurawong-stripe Nice catch, fixed!